generate-changelog.js

```diff
@@ -0,0 +1,86 @@
+const { program } = require('commander');
```

Comment:
[clarification] Is this script necessary because semantic-release's changelog plugin doesn't generate a CHANGELOG file for existing releases/commits, adding entries only for new releases?

Reply:
This was confirmed during a meeting.